code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -1,20 +1,35 @@
 //All defines used in reactions are located in ..\__DEFINES\reactions.dm
-/*priority so far, check this list to see what are the numbers used. please use a different priority if the reaction doesn't work (higher number are done first)
+/*priority so far, check this list to see what are the numbers used. Please use a different priority for each reaction(higher number are done first)
 miaster = -10 (this should always be under all other fires)
-freonfire = -4
-plasmafire = -3
-h2fire = -2
-tritfire = -1
+freonfire = -5
+plasmafire = -4
+h2fire = -3
+tritfire = -2
+halon_o2removal = -1
 nitrous_decomp = 0
 water_vapor = 1
-fusion = 2
+pluox_formation = 2
```

Comment:
since i added many reactions i wanted to be sure that the fusion one wouldn't be affected too much from the new ones mostly because of common gases used in there.

Reply:
:)